### PR TITLE
Gui dev

### PR DESCRIPTION
**Pull request by gui-brandalley**

### CODE REVIEW

File: dbt/brandalley-dbt/models/customer_service/zendesk/_docs.yml [hunk 1]
```diff
@@ -383,4 +383,309 @@ models:
...
+            - unique
+            - not_null 
+        meta: 
+          metrics:
+            total_customers_with_subsequent_order:
+              label: 'Customers with subsequent order'
+              type: count_distinct
+              description: "Number of customers who have returned to make another order"
+              hidden: false
+              sql: "if((${status} in 'solved', closed) and total_agent_replies<=2, ${ticket_id}, null)"
```
[the remark this answers is not in the document]
Comment (nicklupu) [reply dbt/brandalley-dbt/models/customer_service/zendesk/_docs.yml:469]: and does total_agent_replies need to be wrapped in ${}

[the remark this answers is not in the document]
Comment (nicklupu) [reply dbt/brandalley-dbt/models/customer_service/zendesk/_docs.yml:469]: is this the only metrics we need additionally that has meant needing to clone the whole file? 🤔

File: dbt/brandalley-dbt/models/customer_service/zendesk/daily_zendesk_call.sql [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
-    time(timestamp_seconds(max(IF(completion_status!='completed', duration, 0))))                          as max_abandoned_duration
+    date(safe_cast(created_at as datetime))                                                                                      as created_at,
+    count (distinct id)                                                                                                          as number_of_calls,
+    count (IF(direction='inbound', id, null))                                                                                    as inbound_calls,
+    count (IF(direction='outbound', id, null))                                                                                   as outbound_calls,
+    count (IF(completion_status='completed', id, null))                                                                          as completed_calls,
+    count (IF(completion_status!='completed', id, null))                                                                         as abandoned_calls,
+    cast(time(timestamp_seconds(sum(duration))) as string)                                                                       as call_duration,
```
Comment (nicklupu) [inline dbt/brandalley-dbt/models/customer_service/zendesk/daily_zendesk_call.sql:8]: how comes so much casting?

**Review by nicklupu — CHANGES_REQUESTED** [review]

File: dbt/brandalley-dbt/models/customer_service/zendesk/_docs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gui-brandalley) [reply dbt/brandalley-dbt/models/customer_service/zendesk/_docs.yml]: No, there will be more to come.

File: dbt/brandalley-dbt/models/customer_service/zendesk/daily_zendesk_call.sql (hunk 2 above)
Comment (gui-brandalley) [reply dbt/brandalley-dbt/models/customer_service/zendesk/daily_zendesk_call.sql:8]: duration is in minutes, the time(timestamp_seconds) helps to put it in HH:mm:SS format but in LD this format is not read properly hence the cast to string.

**Review by nicklupu — APPROVED** [review]

**Pull request merged**